recipes/chunkio/all/patches/1.1.6-0003-disable-pic.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ebf3c4c..2db2794 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -7,7 +7,9 @@ set(CIO_VERSION_PATCH  6)
+ set(CIO_VERSION_STR "${CIO_VERSION_MAJOR}.${CIO_VERSION_MINOR}.${CIO_VERSION_PATCH}")
+ 
+ # CFLAGS
+-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
++if(BUILD_SHARED_LIBS)
++  set(CMAKE_POSITION_INDEPENDENT_CODE ON)
++endif()
```

Comment:
Should we remove the `fpic` option from the Conan recipe?

Reply:
IMO, I think that Conan should manage it through the option. Wdyt? @RubenRBS @SpaceIm @uilianries

Reply:
I'm fine with this change. fPIC management is more or less regular in CCI, to avoid cases where we build a package without fPIC and try to link to another package with fPIC.